UnitsNet.Tests/DecimalOverloadTests.cs

```diff
@@ -18,7 +18,7 @@ public static void CreatingQuantityWithDoubleBackingFieldFromDecimalReturnsCorre
         public static void CreatingQuantityWithDecimalBackingFieldFromDecimalReturnsCorrectValue()
         {
             Power power = Power.FromWatts(1m);
-            Assert.Equal(1.0m, power.Watts);
+            Assert.Equal(1.0, power.Watts);
```

Comment:
Should change this to Information to keep test's intent of decimal-backed quantity

Reply:
Never mind, this is out of scope for this PR, see below comment on fixing up these old tests.

Reply:
I removed these